internal/pkg/backend/backend.go

```diff
@@ -4,21 +4,12 @@ import (
 	"fmt"
 
 	"github.com/merico-dev/stream/internal/pkg/backend/local"
+	"github.com/merico-dev/stream/pkg/util/log"
 )
 
-type BackendType string
+type Type string
```

Comment:
the `BackendType` and `BackendLocal` below start with the package name isn't a good name style. Because the users will write the code like: `backend.BackendType` or backend.BackendLocal`. Obviously the `backend.Type & backend.Local` is better.